securedrop/journalist.py

```diff
@@ -188,6 +189,9 @@ def admin_add_user():
                 form_valid = False
                 flash("Your password is too long (maximum length {} characters)".format(
                         Journalist.MAX_PASSWORD_LEN), "error")
+            except WeakPasswordError:
+                form_valid=  False
+                flash("Your password is too weak. Try adding more words." 'error')
```

Comment:
Typo (missing comma): ` flash("Your password is too weak. Try adding more words.", 'error')`